tools/mpremote/mpremote/console.py

```diff
@@ -96,8 +96,19 @@ def inWaiting(self):
         return 1 if self.ctrl_c or msvcrt.kbhit() else 0
 
     def waitchar(self, pyb_serial):
-        while not (self.inWaiting() or pyb_serial.inWaiting()):
-            time.sleep(0.01)
+        import serial
```

Comment:
So that we don't need to rely on the `serial` package (eg maybe `pyb_serial` is not a `Serial` instance), I suggest not doing this import and then catching the general `OSError` below, instead of catching `serial.SerialException`.